.github/workflows/maven.yml

```diff
@@ -9,8 +9,6 @@ on:
       - main
       - /.*-stable/
   pull_request:
-    branches:
```

Comment:
We should run the build for any PR. Currently, it won't run for example if we are targeting `release/x.y.z`